doc/rbd/rbd-snapshot.rst

```diff
@@ -66,75 +73,94 @@ Create Snapshot
 ---------------
 
 To create a snapshot with ``rbd``, specify the ``snap create`` option, the pool
```

Comment:
`snap create` is not an option, it's a subcommand.  An example of an option is `--user` or `--no-progress`.  In another PR, we settled on not splitting the command (`rbd`) and the subcommand (`snap create`), so I'd phrase this as "To create a snapshot, use `rbd snap create` command and specify ...".

Reply:
Done.